src/Data/Sum/Properties.agda

```diff
@@ -58,6 +62,22 @@ swap-involutive = [ (λ _ → refl) , (λ _ → refl) ]
 
 map-commute : {f : A → B}  {g : C → D}
               {f′ : B → E} {g′ : D → F} →
-              ((map f′ g′) ∘ (map f g)) ≗ map (f′ ∘ f) (g′ ∘ g)
+              (map f′ g′) ∘ (map f g) ≗ map (f′ ∘ f) (g′ ∘ g)
```

Comment:
Those brackets were pointless so I removed them

Reply:
The ones around `map` probably are too.